### DISCUSSION TITLE
I want to see my own releases in my feed

### DISCUSSION THREAD

**Discussion by sacredbanana** [community/community#113131] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I make a release in my repos I am not seeing the release appear in my feed even though I have starred my own repo. I want to see it in my feed so I can quickly see people thumbing it up. It seems odd that it appears on everyone else's feed except the person who actually owns the repo

**Comment by IvankaKHan** [comment]
Hi

**Comment by Rawknee-69** [comment]
Well here is what you can do to fix it and if you find this helpful please mark this as an answer @sacredbanana 

1.Check Your GitHub Notifications Settings: Ensure that you're subscribed to releases for the repository.
Go to the repository page, click on the "Watch" button at the top right, and select "Watching" to get notifications for releases and other activities.

2.Ensure You're Logged In to the Correct Account: Sometimes, GitHub can show different behavior if you're logged in with an account that is not the one you expect to receive notifications for.

3.Star the Repository from a Different Account: If you're testing the feed behavior with your own account, try starring the repository from a different account (e.g., a secondary or test account) to see if the release appears in that feed.

4.Check for Any Custom Filters: On your GitHub homepage (the feed), check if you’ve applied any custom filters or muted certain types of activities that might hide your own repository updates. Make sure the feed is set to show all activities.

5.Use GitHub Mobile App: Sometimes the web interface doesn't show all updates in the feed. You might see the release in your feed on the GitHub Mobile App, which can give you more real-time updates and a fresh perspective.
